core/modules/wishlist/helpers/notifications.ts

```diff
@@ -0,0 +1,21 @@
+import i18n from '@vue-storefront/i18n'
```

Comment:
please move all these notifications into the theme to be easily changed and managed